File: Phasegate/Day Four/CheckOutApp.java
import java.util.Scanner;
public class CheckoutApp{
    public static void main(String[] args){
        Scanner yomi = new Scanner(System.in);
        String[] storeDetails = {
            "SEMICOLON STORES",
            "MAIN BRANCH",
            "LOCATION: 312, HERBERT MACAULAY WAY, SABO YABA, LAGOS.",
            "TEL: 03293828343"
        };
        for (int index = 0; index < storeDetails.length; index++){
            System.out.println(storeDetails[index]);
        }
        System.out.println("==================================================");
        System.out.print("Cashier Name: ");
        String cashierName = yomi.nextLine();
        System.out.print("Customer Name: ");
        String customerName = yomi.nextLine();
        System.out.print("How many products does the customer have?: ");
        int numberOfProducts = yomi.nextInt();
        String[] productNames = new String[numberOfProducts];
        int[]    quantities   = new int[numberOfProducts];
        double[] prices       = new double[numberOfProducts];
        double[] itemTotals   = new double[numberOfProducts];        
        for (int index = 0; index < numberOfProducts; index++){
            System.out.println("--- Product " + (index + 1) + " ---");
            System.out.print("Product Name: ");
            productNames[index] = yomi.nextLine();
            System.out.print("Price: ");
            prices[index] = yomi.nextInt();
            System.out.print("Quantity: ");
            quantities[index] = yomi.nextInt();
            itemTotals[index] = prices[index] * quantities[index];
        }
        double subTotal = 0;
        for (int index = 0; index < numberOfProducts; index++){
            subTotal = subTotal + itemTotals[index];
        }
        double discount = 0;
        if (subTotal > 5000){
            discount = subTotal * 0.08;
        }
        double vat = (subTotal - discount) * 0.075;
        double billTotal = subTotal - discount + vat;
        System.out.print("How much did the customer give to you?: ");
        double amountPaid = yomi.nextInt();
        double balance = amountPaid - billTotal;
        System.out.println("==================================================");
        for (int invoice = 0; invoice < storeDetails.length; invoice++){
            System.out.println(storeDetails[invoice]);
        }
        System.out.println("Cashier: " + cashierName);
        System.out.println("Customer Name: " + customerName);
        System.out.println("==================================================");
        System.out.println("ITEM  QTY    PRICE      TOTAL");
        System.out.println("--------------------------------------------------");
        for (int invoice = 0; invoice < numberOfProducts; invoice++) {
            System.out.println(productNames[invoice] + "   " + quantities[invoice] + "   " + prices[invoice] + "   " + itemTotals[invoice]);
        }
        System.out.println("--------------------------------------------------");
        System.out.println("Sub Total:     " + subTotal);
        System.out.println("Discount:      " + discount);
        System.out.println("VAT @ 7.5%:    " + vat);
        System.out.println("==================================================");
        System.out.println("Bill Total:    " + billTotal);
        System.out.println("Amount Paid:   " + amountPaid);
        System.out.println("Balance:       " + balance);
        System.out.println("=====================================================================");
        System.out.println("     THANK YOU FOR YOUR PATRONAGE YOMI GREETS AND COME AGAIN");
        System.out.println("=============================================================e");
    }
}

File: Phasegate/Day Four/check_out_app.py
store_details = [
    "SEMICOLON STORES",
    "MAIN BRANCH",
    "LOCATION: 312, HERBERT MACAULAY WAY, SABO YABA, LAGOS.",
    "TEL: 03293828343"
]
for index in range(len(store_details)):
    print(store_details[index])
print("==================================================")
cashier_name  = input("Cashier Name: ")
customer_name = input("Customer Name: ")
number_of_products = int(input("How many products does the customer have?: "))
product_names = []
quantities    = []
prices        = []
item_totals   = []
for index in range(number_of_products):
    print("--- Product " + str(index + 1) + " ---")
    name  = input("Product Name: ")
    price = float(input("Price: ")
    qty   = int(input("Quantity: "))
    product_names.append(name)
    prices.append(price)
    quantities.append(qty)
    item_totals.append(price * qty)
sub_total = 0
for index in range(len(item_totals)):
    sub_total = sub_total + item_totals[index]
discount = 0
if sub_total > 5000:
    discount = sub_total * 0.08
vat        = (sub_total - discount) * 0.075
bill_total = sub_total - discount + vat
amount_paid = float(input("How much did the customer give to you?: "))
balance     = amount_paid - bill_total
print("==================================================")
for index in range(len(store_details)):
    print(store_details[index])
print("Cashier: " + cashier_name)
print("Customer Name: " + customer_name)
print("==================================================")
print("ITEM              QTY    PRICE      TOTAL(NGN)")
print("--------------------------------------------------")
for index in range(number_of_products):
    print(product_names[index] + "   " + str(quantities[index]) + "   " + str(prices[index]) + "   " + str(item_totals[index]))
print("--------------------------------------------------")
print("Sub Total:     " + str(sub_total))
print("Discount:      " + str(discount))
print("VAT @ 7.5%:    " + str(vat))
print("==================================================")
print("Bill Total:    " + str(bill_total))
print("Amount Paid:   " + str(amount_paid))
print("Balance:       " + str(balance))
print("==================================================")
print("     THANK YOU FOR YOUR PATRONAGE")
print("==================================================")
